### PR TITLE
Adds the syndicate maid kit

### PR DESCRIPTION
**Pull request by Maia-J**
## About The Pull Request

Adds a syndicate maid kit, a 8 TC badass kit that's basically the same stats as the redsuit mod except no space protection and no suit wear slot.

Contains:
Tactical maid headband
Tactical maid outfit (Blocks wearing anything in the suit slots to avoid armor stacking)
Tactical maid arm covers
High heeled boots (Anti slip, in line with redsuit)

Also moves some files in one place

## Why It's Good For The Game

It is cool, funny, and I was asked to do it

## Proof Of Testing

I did test all kinds of removing the clothes and checking the signals and shit it should be clear

## Changelog

:cl:
add: Syndicate maid kit, a kit containing armored syndicate maid clothing
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]

**Comment by Odairu** [comment]
If its the same stats, but otherwise entirely worse, why not make it cost less

**Comment by Swiftfeather** [comment]
> If its the same stats, but otherwise entirely worse, why not make it cost less

It's for pointless badassery

**Comment by Elli-Skala** [comment]
haha, yeah badassery, yeah

**Review by Swiftfeather — APPROVED** [review]

File: modular_zubbers/code/modules/uplink/uplink_items/syndimaid/clothes.dm [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	bullet = 50
+	energy = 40
+	fire = 50
+	laser = 30
+	melee = 40
+	wound = 25
+
+/obj/item/clothing/under/syndicate/skyrat/maid/armored
+	desc = "A 'tactical' skirtleneck fashioned to the likeness of a maid outfit.\
+	 The armor on this one is too bulky blocks the user from wearing any chest armor."
```
Comment (nikothedude) [inline modular_zubbers/code/modules/uplink/uplink_items/syndimaid/clothes.dm:15]: > ...one is bulky and blocks...

reads weird otherwise

**Comment by NargaInDisguise** [comment]
Redsuits actually don't have antislip built into them, funny enough, so technically this is a sidegrade but then we get into a whole arguement about adding mods and if space protection is better (it is) so ???

It is a funny kit, I got nothing wrong with it.

**Comment by theselfish** [comment]
My dumbass must have not seen the uplink folder in modular, so you may wanna move my dumb stuff in;
https://github.com/Bubberstation/Bubberstation/tree/master/modular_zubbers/code/modules/antagonists/uplink

To the files you're using.

**Comment by Majkl-J** [comment]
> To the files you're using.

I just shoved it all into the antag folder. See https://github.com/Bubberstation/Bubberstation/pull/2024/commits/2b0f094022c8986418c1b6226bd912609a2c2d44

**Comment by Fluffy-kitt** [comment]
> Redsuits actually don't have antislip built into them, funny enough, so technically this is a sidegrade but then we get into a whole arguement about adding mods and if space protection is better (it is) so ???
> 
> It is a funny kit, I got nothing wrong with it.

Redsuits get knockdown from baton immunity inbuilt; atop of space prot and mods

**Pull request merged**